psocake/gui.py

```diff
@@ -491,6 +567,20 @@ def mouseClicked(evt):
 
 def main():
     global ex
+    global eventDocument
+    eventDocument = {}
+    global hit_dict
+    hit_dict = { QtCore.Qt.Key_4 : 0,   # No     hit (alias)
```

Comment:
No hit (alias) is confusing. Why is this needed?

Reply:
Key 0 is on the far right of the keyboard.  The alias allows a user to keep his/her fingers at Key 1-4 area.   This feature is not necessary for Numpad users.